runexporterapp.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export IRONIC_CONFIG=/etc/ironic/ironic.conf
+export FLASK_RUN_HOST=0.0.0.0
```

Comment:
We need to be able to control this from the outside of the container, so that ironic only binds to the provisioning network interface.

Reply:
can this be done in a follow-up?

Reply:
Sure. Would you open an issue to track that?

Reply:
Isn’t this just a trivial one line change to accept an env var?